plugin/src/main/java/org/screamingsandals/bedwars/special/ArrowBlocker.java

```diff
@@ -68,7 +69,11 @@ public void activate() {
             if (item.getAmount() > 1) {
         		item.setAmount(item.getAmount() - 1);
         	} else {
-        		player.getInventory().remove(item);
+        		if(player.getInventory().getItemInOffHand().equals(item)) {
```

Comment:
Add try-catch blocks here for Throwables, because of older versions. Don't print stacktrace, instead of put in the catch block ```player.getInventory().remove(item);```

Reply:
Ok I added the try-catch

Reply:
Now the format :P